corehq/apps/accounting/async_handlers.py

```diff
@@ -161,14 +161,15 @@ def country_response(self):
         from django_countries.data import COUNTRIES
         countries = sorted(COUNTRIES.items(), key=lambda x: x[1].encode('utf-8'))
         if self.search_string:
-            return filter(lambda x: x[1].lower().startswith(self.search_string.lower()), countries)
+            search_string = self.search_string.capitalize()
```

Comment:
You are relying on the fact that "All the countries are Capitalised" which may be true, but if they upgrade and change that, or if some new country doesn't capitalize for some weird reason, etc. then it would break. by coercing both to the same case you are making it more future-proof of any unexpected changes.